src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -861,12 +863,6 @@ li {
     color: $gray9;
   }
 
-  // align top of text with checkbox icons
-  .v-input--checkbox .v-label {
-    padding-top: 2px;
-  }
-}
```

Comment:
This doesn't look right: you deleted 1 open-curly-brace but deleted 2 close-curly-braces.

Reply:
Yep. My bad.